packages/messenger-widget/src/components/ConfigureProfile/context/ConfigureProfileContext.tsx

```diff
@@ -43,15 +44,36 @@ export const ConfigureProfileContextProvider = (props: { children?: any }) => {
 
     const [errorMsg, setErrorMsg] = useState<string>('');
 
+    const { disabledOptions } = useContext(ModalContext);
+
     // DM3 Name service selected
     const [dm3NameServiceSelected, setDm3NameServiceSelected] =
-        useState<string>(dm3NamingServices[0].name);
+        useState<string>(
+            dm3NamingServices.filter(
+                (d) =>
```

Comment:
either extract that logic to a function, or add comments explaining what is happening here